users/shared_account/shared_account.go

```diff
@@ -0,0 +1,157 @@
+//   Copyright 2018 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package shared_account
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/trackit/trackit-server/routes"
+	"github.com/trackit/trackit-server/users"
+	"github.com/trackit/trackit-server/db"
+)
+
+// inviteUserRequest is the expected request body for the invite user route handler.
+type InviteUserRequest struct {
+	Email           string `json:"email" req:"nonzero"`
+	AccountId       int    `json:"accountId"`
+	PermissionLevel int    `json:"permissionLevel"`
+}
+
+type listUserSharedAccountRequest struct {
+	AccountId       int    `json:"accountId" req:"nonzero"`
```

Comment:
Wouldn't it be more convenient to return all shared accounts without requiring an accountId?

Reply:
This is not to return a list of shared accounts but to return users who have an access to a specific aws_account in Trackit.